hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
 const elem = document.querySelector('input');
+const result = document.querySelector('div > div');
 
 elem.addEventListener('input', handleInput);
+
+function handleInput() {
```

Comment:
To adhere to the Airbnb Style Guide. [19.1](https://github.com/airbnb/javascript#whitespace--spaces) states we should use soft tabs(2 spaces instead of 4).